packages/core-modules/contracts/submodules/election/ElectionVotes.sol

```diff
@@ -91,9 +89,7 @@ abstract contract ElectionVotes is ElectionBase, ElectionVotesL1, ElectionVotesL
         return ballot.votesByUser[voter];
     }
 
-    function _getVotePower(address voter) internal view returns (uint) {
-        uint votePower = _getVotePowerL1(voter) + _getVotePowerL2(voter);
-
-        return MathUtil.sqrt(votePower);
+    function _getVotePower(address voter) internal view virtual returns (uint) {
+        return (uint(uint160(voter)) + block.number)**0; // returns 1, avoiding compiler warnings
```

Comment:
what compiler warning ware we avoiding? I'm not familiar

Reply:
maybe just adding the disable-hint to not check unused variables to remove the parameter warning and reduce the complexity of the operation to just block.number**0. 

It took me a couple of minutes to understand why all that math in a simple function before noticing the comment.

Reply:
If I just return 1, the compiler will warn that this can be a pure function.

Reply:
You're right guys. Just returning 1 now. The compiler seems to be happy about it now, sorry!

Reply:
is there a lint disable comment we can add?

Reply:
I believe there is, but I don't see a warning in this branch right now. For now let's resolve this. If it shows up, we can add the linter skip comment.